asserts/validation_set.go

```diff
@@ -176,6 +176,14 @@ type ValidationSet struct {
 	timestamp time.Time
 }
 
+func (vs *ValidationSet) Key() string {
```

Comment:
there are too many things Key can refer to in the context of assertions, als snapasserts.ValidationSetKey wraps the full primary key while this return only sequence key but without the series. I think we need a different name for this? SequenceName?

Reply:
this also in all cases needs a doc comment

Reply:
another approach would be to include the series, then it would be SequenceKey but I don't know how it affects the places using this